test/test-tables/pool_units.json

```diff
@@ -1,4 +1,12 @@
 {
+  "_info": {
```

Comment:
Ha nice idea. I hate how you cannot have comments in json.

Reply:
haha credit goes to @Jedi2002